apps/flutter_exercise/lib/views/widgets/login_widgets/form.dart

```diff
@@ -0,0 +1,231 @@
+  //
+  // Note: This is a GlobalKey<FormState>,
+  // not a GlobalKey<MyCustomFormState>.
+  final _formKey = GlobalKey<FormState>();
+
+  @override
+  Widget build(BuildContext context) {
+    // Build a Form widget using the _formKey created above.
+    return Container(
+      margin: const EdgeInsets.only(left: 40, right: 40, top: 100),
+      child: Form(
+        key: _formKey,
+        child: Column(
+          crossAxisAlignment: CrossAxisAlignment.center,
+          children: [
+            const Align(
+              alignment: Alignment.topRight,
+              child: Text(
+                "Ayuda",
+                textAlign: TextAlign.right,
+                style: TextStyle(
+                    color: Color.fromARGB(255, 23, 57, 25),
+                    decoration: TextDecoration.underline),
+              ),
+            ),
+            Container(
+              margin: const EdgeInsets.only(left: 70, right: 70, bottom: 20),
+              child: const Column(
+                children: [
+                  Text(
+                    "Regístrate",
+                    style: TextStyle(
+                      color: Colors.orangeAccent,
+                      fontWeight: FontWeight.bold,
+                      fontSize: 32,
+                    ),
+                  ),
+                  Padding(
+                    padding: EdgeInsets.only(top: 10),
+                  ),
+                  Text("Pon el marcapáginas, solo nos llevará unos segundos",
+                      textAlign: TextAlign.center,
+                      style: TextStyle(
+                        color: Color.fromARGB(255, 23, 57, 25),
+                        fontWeight: FontWeight.bold,
+                      ))
+                ],
+              ),
+            ),
+            _fieldForm("nombre"),
+            _fieldForm("email"),
+            _fieldForm("contraseña"),
+            const Align(
+              alignment: Alignment.topRight,
+              child: Padding(
+                padding: EdgeInsets.only(right: 8),
+                child: Text(
+                  "8 caracteres mínimo",
+                  textAlign: TextAlign.right,
+                  style: TextStyle(
+                    fontSize: 10,
+                    color: Color.fromARGB(255, 255, 0, 0),
+                  ),
+                ),
+              ),
+            ),
+            _fieldForm("contraseña", "Confrima tu "),
+            const Align(
+              alignment: Alignment.topRight,
+              child: Padding(
+                padding: EdgeInsets.only(right: 8),
+                child: Text(
+                  "8 caracteres mínimo",
+                  textAlign: TextAlign.right,
+                  style: TextStyle(
+                    fontSize: 10,
+                    color: Color.fromARGB(255, 255, 0, 0),
+                  ),
+                ),
+              ),
+            ),
+            const Row(mainAxisAlignment: MainAxisAlignment.center, children: [
```

Comment:
@Mazit0 Is better put any property in a single line.
const Row(
     mainAxixAlignment: ...
     children: [
         ...
     ]